extensions/grpc/deployment/src/test/java/io/quarkus/grpc/server/services/BlockingMutinyTestService.java

```diff
@@ -32,13 +36,15 @@
     public Uni<EmptyProtos.Empty> emptyCallBlocking(EmptyProtos.Empty request) {
         assertThat(request).isNotNull();
         assertRunOnWorker();
+        assertRunOnDuplicatedContext();
```

Comment:
I'm a bit surprised that not being on the event loop we still have a vertx context

Reply:
Yes, I know it's confusing. But inside an "executeBlocking" block, you still have access to the same context as outside (leading to fun concurrency issue)

Reply:
does it work for the root contexts too? Or only duplicated ones?